eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -133,11 +133,11 @@ if ($relatedCadlProjectFolder) {
             throw "Not provide service name or namespace."
         }
         $projectFolder = (Join-Path $sdkPath "sdk" $service $namespace)
-        New-CADLPackageFolder `
+        New-TypeSpecPackageFolder `
             -service $service `
             -namespace $namespace `
             -sdkPath $sdkPath `
-            -relatedCadlProjectFolder $cadlRelativeFolder `
+            -relatedTypeSpecProjectFolder $cadlRelativeFolder `
```

Comment:
```suggestion
            -relatedTypeSpecProjectFolder $typespecRelativeFolder `
```

Reply:
No. This is from Unified pipeline. Current Unified pipeline has not update the parameter name, so we will not update for consistent at this time. After Unified pipeline updated the parameter name, we will update.